hngx-org/team-artemis-breakout
# Aj kenz

## Pull Request Description

### Description

This pull request introduces several enhancements to our leaderboard feature. It includes updates to the `BoardItems` component and the `LeaderBoard` component, as well as improvements to the data management.

### Changes Made

-  Built the UI for the LeaderBoard Screen
- Added a `rank` property to the `BoardItems` component, which is now displayed for each player.
- Updated the `LeaderBoard` component to sort the player data based on their scores and assign ranks.
- Improved the data management in the `LeaderBoard` component for a more dynamic leaderboard.

### How to Test

1. Clone this repository and checkout this pull request branch.
2. Run the application.
3. Navigate to the leaderboard feature.
4. Verify that the player data is displayed with the corresponding ranks based on their scores.

### Checklist

- [x] Code follows our coding standards and guidelines.
- [x] Documentation (such as this pull request description) has been updated to reflect the changes made.
